libjsonutils/src/json_utils.cpp

```diff
@@ -1,23 +1,24 @@
+#include "file_utils.h"
+
 #include "jsonutils/json_utils.h"
 
 #include <boost/regex.hpp>
 #include <iostream>
 
-#include <file_utils.h>
 
 namespace jsonutils
 {
     //Extremely poor regex pattern for matching urls
     constexpr auto URL_PATTERN = "(http|https)://(\\w+\\.)+(\\w)/?(\\w+/{0,1})*";
 
-    boost::optional<rapidjson::Document> loadFromUrl(const std::string& url)
+    boost::optional<rapidjson::Document> loadFromUrl(const std::string& /* url */)
     {
         //Download resource and extract message body
         //Return boost::none if something went wrong, e.g. status code >= 300
         rapidjson::Document doc;
         doc.Parse("{\"source\": \"url\"}");
 
-        return doc;
+        return std::move(doc);
```

Comment:
Never `std::move()` a local variable http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f48-dont-return-stdmovelocal
It may inhibit RVO
```suggestion
        return doc;
```